Britannia/localization/english/modifiers/BTWK_modifiers_l_english.yml

```diff
@@ -0,0 +1,37 @@
+﻿l_english:
+
+## CHARACTER MODIFIERS ##
```

Comment:
Yes. Visual Studio allows me to collapse sections indented within #s. Given how unreadable vanilla loc files are, I find this better for navigation (especially if you don't know the name of what you're looking for). I can revert it back to vanilla loc file style if you like though. Doesn't impact anything either way - the error logs haven't thrown any hissy fits about it I'm aware of.

Reply:
Ah okay, that's fine then. Just one request, could you adjust the indentation so that the first level of comments are still under "l_english:" ? (i.e. just increase the indentation level of everything by one)